contracts/misc/ZeroReserveInterestRateStrategy.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import {DataTypes} from '../protocol/libraries/types/DataTypes.sol';
+import {IDefaultInterestRateStrategy} from '../interfaces/IDefaultInterestRateStrategy.sol';
+import {IReserveInterestRateStrategy} from '../interfaces/IReserveInterestRateStrategy.sol';
+import {IPoolAddressesProvider} from '../interfaces/IPoolAddressesProvider.sol';
+
+/**
+ * @title ZeroReserveInterestRateStrategy contract
+ * @author Aave
+ * @notice Interest Rate Strategy contract, with all parameters zeroed.
+ * @dev It returns zero liquidity and borrow rate.
+ */
+contract ZeroReserveInterestRateStrategy is IDefaultInterestRateStrategy {
+  /// @inheritdoc IDefaultInterestRateStrategy
+  uint256 public immutable OPTIMAL_USAGE_RATIO = 0;
```

Comment:
Why `immutable` and just not `constant`?

Reply:
good catch, fixed now